src/Lean/Linter/MissingDocs.lean

```diff
@@ -19,111 +17,176 @@ register_builtin_option linter.missingDocs : Bool := {
 
 def getLinterMissingDocs (o : Options) : Bool := getLinterValue linter.missingDocs o
 
+
+namespace MissingDocs
+
+abbrev SimpleHandler := Syntax → CommandElabM Unit
+abbrev Handler := Bool → SimpleHandler
```

Comment:
```suggestion
abbrev SimpleHandler ks := TSyntax ks → CommandElabM Unit
abbrev Handler ks := Bool → SimpleHandler ks
```
Then we could also drop the `foo` in `@[missingDocsHandler foo]`.

Reply:
That makes the `evalConst` a lot trickier, I think?

Reply:
Yeah, this is an interesting general refactoring that we can apply to other places as well, especially after strengthening the macro typing story.